test_programs/execution_success/brillig_fns_as_values/src/main.nr

```diff
@@ -7,9 +7,9 @@ struct MyStruct {
 fn main(x: u32) {
     assert(wrapper(increment, x) == x + 1);
     assert(wrapper(increment_acir, x) == x + 1);
-    assert(wrapper(decrement, x) == std::wrapping_sub(x, 1));
+    assert(wrapper(decrement, x) == x - 1);
     assert(wrapper_with_struct(MyStruct { operation: increment }, x) == x + 1);
-    assert(wrapper_with_struct(MyStruct { operation: decrement }, x) == std::wrapping_sub(x, 1));
+    assert(wrapper_with_struct(MyStruct { operation: decrement }, x) == x - 1);
```

Comment:
A wrapping sub should have different behavior than a non-wrapping one it was replaced with here.
Why was `x` changed from 0 to 1 so that we don't test wrapping anymore?
Maybe we can do a middle ground where we test both `x - 1` and `wrapping_sub(x, 1)` since the new failure case is tested in brillig_overflow_checks

Reply:
This test wasn't supposed to test wrapping/checked just using functions as values. It was modified before to add wrapping_sub when overflow checks were added for acir because the test was doing wrapping subtractions since the value passed was 0, this restores the test to its original form. I'll add an explicit brillig wrapping test!